### PR TITLE
(#2584) Remove license leeway comments

### LINKED ISSUE #2584: License add warning if expired
As noted in a todo comment in this file: https://github.com/chocolatey/choco/blob/cbba584692e2d02012e3eb03ba366b63d741d110/src/chocolatey/infrastructure/licensing/LicenseValidation.cs#L129-L130

### PR DESCRIPTION
**Pull request by pauby**
Removes comments that suggest there is a warning provided when a license expires.

Closes #2584

### CODE REVIEW

File: src/chocolatey/infrastructure/licensing/LicenseValidation.cs [hunk 1]
```diff
@@ -126,8 +126,6 @@ public static ChocolateyLicense validate()
                 chocolateyLicense.Name = license.Name;
                 chocolateyLicense.Id = license.UserId.to_string();
```
Comment (JPRuskin) [inline src/chocolatey/infrastructure/licensing/LicenseValidation.cs:128]:
```suggestion
```

Personal preference to remove the blank line as well, fine if we don't want to.

**Review by JPRuskin — APPROVED** [review]

File: src/chocolatey/infrastructure/licensing/LicenseValidation.cs (hunk 1 above)
Comment (pauby) [reply src/chocolatey/infrastructure/licensing/LicenseValidation.cs:128]: I thought about that but seeing the rest of the code I thought it was more consistent to leave it. But looking at it again I think it makes sense. I'll remove it.

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@pauby thanks for getting this fixed up!

**Pull request merged**